packages/deployment/lib/kes.js

```diff
@@ -155,7 +161,9 @@ class UpdatedKes extends Kes {
     const src = path.join(process.cwd(), kesBuildFolder, filename);
     const dest = path.join(process.cwd(), kesBuildFolder, 'adapter', unzipFolderName);
 
+    // If not using message adapter, skip custom compilation
```

Comment:
Just to double check, it would appear that if a lambda isn't configured to run with CMA, then it won't have access to the versioning capabilities. Is that right?

Reply:
@ifestus  I think we talked about this "in-person", but that's not quite right - this flag is a template level flag.